docs/multicluster/policy-only-mode.md

```diff
@@ -0,0 +1,83 @@
+# Antrea Multi-cluster with NetworkPolicy Only Mode
+
+Multi-cluster Gateway works with Antrea `networkPolicyOnly` mode, in which
+cross-cluster traffic is routed by Multi-cluster Gateways of member clusters,
+and the traffic goes through Antrea overlay tunnels between Gateways and local
+cluster Pods. Pod traffic within a cluster is still handled by the primary CNI,
+not Antrea.
+
+## Deploying Antrea in `networkPolicyOnly` mode with Multi-cluster feature
+
+This section describes steps to deploy Antrea in `networkPolicyOnly` mode
+with the Multi-cluster feature enabled on an EKS cluster.
+
+Assume you already have an EKS cluster, and have the `KUBECONFIG` environment
+variable point to the kubeconfig file of that cluster. You can follow
+[the EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) to create the cluster.
+
+At first, you can follow the [Antrea ESK installation guide](./../eks-installation.md)
+to deploy Antrea to an EKS cluster. Please note there are a few changes required
+by Antrea Multi-cluster. You should set the following configuration parameters in
+ `antrea-agent.conf` of the Antrea deployment manifest to enable the `Multicluster`
+ feature:
+
+```yaml
+antrea-agent.conf: |
+...
+  featureGates:
+...
+    Multicluster: true
+...
+  multicluster:
+    enableGateway: true
+    namespace: "" # Change to the Namespace where antrea-mc-controller is deployed.
+```
+
+Once you deployed Antrea with Multi-cluster feature, Antrea in a single cluster will
+be ready to handle cross-cluster traffic. You should repeat the same steps to deploy
+Antrea for all member clusters in a ClusterSet. Besides antrea CNI deployment, you
+also need to deploy Antrea multi-cluster controller for a ClusterSet. There are some
+prerequisites you should be aware of, eg: Service CIDRs (ClusterIP ranges) must not
```

Comment:
We only have a single prerequisite. If so, we should change to "Make sure the Service CIDRs (ClusterIP ranges) must not overlay among the member clusters".

Reply:
Updated.